cloudbuild.yaml

```diff
@@ -0,0 +1,26 @@
+steps:
+- name: 'gcr.io/cloud-builders/git'
+  args:
+  - 'clone'
+  - 'https://github.com/qupath/qupath.git'
+  - 'qupath'
+- name: 'gcr.io/cloud-builders/git'
+  args:
+  - 'checkout'
+  - 'tags/v0.1.2'
+  dir: 'qupath'
+- name: 'maven:3.6.1-amazoncorretto-8'
```

Comment:
Is there a specific reason for not using : gcr.io/cloud-builders/mvn - as in the example in this file(as an example on how to specify args) :https://cloud.google.com/cloud-build/docs/build-config

Reply:
I can't use this builder 'gcr.io/cloud-builders/mvn' because it doesn't contain a JavaFX package. But 'maven:3.6.1-amazoncorretto-8' contains this package. Without JavaFX, I can't compile extension and QuPath.

Reply:
Ah ok makes sense thanks.